From aaf1ce7cc025b5da9fecfaa392f4bf9858836e16 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 30 Jul 2011 22:49:06 +0800
Subject: [PATCH] Revert "mmc: sdio: Add high speed support to
 sdio_reset_comm()"

This reverts commit 25c29cc964ec4d29370e8af7ae3b51f1f6a239ac.
---
 drivers/mmc/core/sdio.c | 23 +----------------------
 1 file changed, 1 insertion(+), 22 deletions(-)

diff --git a/drivers/mmc/core/sdio.c b/drivers/mmc/core/sdio.c
index ac9102487e25..a78b4dc3fd9e 100644
--- a/drivers/mmc/core/sdio.c
+++ b/drivers/mmc/core/sdio.c
@@ -661,28 +661,7 @@ int sdio_reset_comm(struct mmc_card *card)
 			goto err;
 	}
 
-	/*
-	 * Switch to high-speed (if supported).
-	 */
-	err = sdio_enable_hs(card);
-	if (err)
-		goto err;
-
-	/*
-	 * Change to the card's maximum speed.
-	 */
-	if (mmc_card_highspeed(card)) {
-		/*
-		 * The SDIO specification doesn't mention how
-		 * the CIS transfer speed register relates to
-		 * high-speed, but it seems that 50 MHz is
-		 * mandatory.
-		 */
-		mmc_set_clock(host, 50000000);
-	} else {
-		mmc_set_clock(host, card->cis.max_dtr);
-	}
-
+	mmc_set_clock(host, card->cis.max_dtr);
 	err = sdio_enable_wide(card);
 	if (err)
 		goto err;
-- 
2.35.3

